pkg/experimental/rpm.bzl

```diff
@@ -722,12 +722,14 @@ pkg_rpm = rule(
             Must be a form that `rpmbuild(8)` knows how to process.  Current known
             options depend on the `rpmbuild` binary, and may include:
 
+            - "w0.ufdio" (no compression)
```

Comment:
Does it really help to have this list here?
It seems like you have to look at the rpmbuild manual page to understand which you would want,
and you are playing catch-up with future additions.

Reply:
Strictly speaking, no.  Unfortunately, this feature seems to somewhat undocumented what exists of it seems to be hidden beneath some other code.

I'll just link to some example in the RPM source code and leave some breadcrumbs saying where users can find it on their systems.